cosmwasm/verify

```diff
@@ -3,20 +3,37 @@
 set -euo pipefail
 
 usage="Usage:
-  $(basename "$0") [-h] [-n network] <.wasm file> <code id> -- Verify that the deployed on-chain bytecode matches the local object file
+  $(basename "$0") [-h] [-n network] [-c chain] [-w .wasm file] [-i code id] -- Verify that the deployed on-chain bytecode matches the local object file
```

Comment:
This script is referenced here: https://github.com/wormhole-foundation/wormhole/blob/551972d74da30fa7fee8cf016351089a03e82586/scripts/contract-upgrade-governance.sh#L449-L467 so we should update that script too.

Reply:
Updated